.buildkite/run-xpu-test.sh

```diff
@@ -12,5 +12,21 @@ remove_docker_container() { docker rm -f xpu-test || true; }
 trap remove_docker_container EXIT
 remove_docker_container
 
-# Run the image and launch offline inference
-docker run --network host --name xpu-test --device /dev/dri -v /dev/dri/by-path:/dev/dri/by-path --entrypoint="" xpu-test python3 examples/offline_inference.py
+# Run the image and test offline inference/tensor parallel
+docker run -it -d --name xpu-test --device /dev/dri -v /dev/dri/by-path:/dev/dri/by-path xpu-test /bin/bash
+docker exec xpu-test bash -c "python3 examples/offline_inference.py"
+docker exec xpu-test bash -c "
+wget https://huggingface.co/datasets/anon8231489123/ShareGPT_Vicuna_unfiltered/resolve/main/ShareGPT_V3_unfiltered_cleaned_split.json
+python3 benchmarks/benchmark_throughput.py \
+  --backend=vllm \
+  --dataset=./ShareGPT_V3_unfiltered_cleaned_split.json \
+  --model meta-llama/Llama-3.2-1B-Instruct \
+  --num-prompts=10 \
+  -tp=2 \
```

Comment:
this can take significantly longer time. do we have enough ci resource for it?

Reply:
how about use `offline_inference_cli.py`,  like `python3 examples/offline_inference_cli.py -tp 2`

Reply:
Tested. `offline_inference_cli.py` works and only takes about half a minute. @youkaichao 
```
[W1202 05:43:12.840373301 OperatorEntry.cpp:155] 
…
INFO 12-02 05:43:40 distributed_gpu_executor.py:61] Maximum concurrency for 2048 tokens per request: 1225.41x
Processed prompts: 100%|█████████████████| 4/4 [00:00<00:00,  6.47it/s, est. speed input: 42.05 toks/s, output: 103.52 toks/s]
Prompt: 'Hello, my name is', Generated text: ' Daniel, I am a student at UBC. I graduated from UBC in'

```